csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,106 +5,88 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static List<string> Tasks { get; set; }
```

Comment:
Here you can use the C# auto-initialize property feature
```suggestion
        public static List<string> Tasks = new List<string>();
```

Reply:
Can you please clarify a little bit further? Is this suggestion based in the fact that the list is initialized immediately at the start of the main and that is why the lazy initialization is worthless in this specific case? Or should I always prioritize to initialize the objects at class definition to ensure they are not null if something were to read them first?

I am also curious to know if initializing the property like this:
```suggestion
        public static List<string> Tasks { get; set; } = new List<string>();
```
would be against syntax conventions/practices (I like how it looks :D)

Reply:
@Rovisible in a previous C# version, to initialize a variable you must use the Constructor method. Now we can initialize a variable when we define it, so, this is useful to avoid initialize issues. The `{ get; set; }` code will work but a cleaner implementation could be just have `public static List<string> Tasks = new List<string>();`.